docs/large_model_inference.md

```diff
@@ -1,6 +1,13 @@
 # Serving large models with Torchserve
 
 This document explain how Torchserve supports large model serving, here large model refers to the models that are not able to fit into one gpu so they need be split in multiple partitions over multiple gpus.
+This page is split into the following sections:
+- [How it works](#how-it-works)
```

Comment:
I remember there was some issue with relative links when we go to pytorch.org so might wanna double check this works, if it does feel free to dismiss this comment